docs/user/reconciliation.md

```diff
@@ -0,0 +1,166 @@
+# Reconciliation
+
+This document describes the cluster-version operator's reconciliation logic and explains how the operator applies a release image to the cluster.
+
+## Release image content
+
+```console
+$ mkdir /tmp/release
+$ oc image extract quay.io/openshift-release-dev/ocp-release:4.1.0[-1] --path /:/tmp/release
```

Comment:
I would pin the version so the example is reproducible

Reply:
This is pinned (to the 4.1.0 release).

Reply:
That's right, I understood this as the image index as opposed to the layer index. However I'm still confused by the usage of the relative index in an example. Are you expecting the layers to change?

Reply:
> However I'm still confused by the usage of the relative index in an example. Are you expecting the layers to change?

We want the top/last layer, which has the manifests in it.  The total number of layers may or may not change, but `-1` seems like a more straightforward way to say "the last layer" than `4` (or whatever we'd use if we wanted a positive index).